geoportal/c2cgeoportal_geoportal/scaffolds/create/qgisserver/pg_service.conf.tmpl

```diff
@@ -0,0 +1,6 @@
+[demo_geomapfish]
```

Comment:
May be something more generic.

Reply:
https://github.com/camptocamp/c2cgeoportal/pull/4158 will fix that :-)